README.md

```diff
@@ -163,8 +195,24 @@ I would like to thank everyone who contributed to the organisation of this codeb
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
+<!-- FAQ (optional) -->
+
+## ❓ FAQ (OPTIONAL) <a name="faq"></a>
+
+- **[I cannot clone your project to my local machine ?]**
+
+  - [Please check whether you have or properly installed node.]
+
+- **[The project cannot be displayed in the browser ?]**
+
+  - [Please run the command npm start in your terminal and make sure that you are in the application folder.]
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
 <!-- LICENSE -->
 
 ## 📝 License <a name="license"></a>
 
 This project is [MIT](./LICENSE) licensed.
+
```

Comment:
Everything looks great 👏 , however your MIT link holds a broken link, kindly fix that to make your README file even more professional 👍